msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -480,6 +481,39 @@ void mergeConfiguration(PublicClientApplicationConfiguration config) {
         this.isAuthorizationInCurrentTask = config.isAuthorizationInCurrentTask == null ? this.isAuthorizationInCurrentTask : config.isAuthorizationInCurrentTask;
     }
 
+    void mergeGlobalConfiguration(GlobalSettingsConfiguration globalConfig) {
```

Comment:
nit: add private/final/nonnull

Reply:
I made this protected, since it's being called in `GlobalSettings.java`